packages/webclient/src/Authorizer/AzureEndpoint.ts

```diff
@@ -30,7 +29,6 @@ export class AzureEndpoint implements IAuthorizer {
         this._baseUrl = "https://management.azure.com/";
         this.environmentAuthorityUrl = "https://login.windows.net/";
         this.activeDirectoryResourceId = "https://management.core.windows.net/";
-        this._webClient = new WebClient();
```

Comment:
Can you verify once that webclient is not being accessed anywhere from AzureEndpoint reference?
for safer side don't remove this initialisation